Python/pylifecycle.c

```diff
@@ -2204,7 +2204,7 @@ fatal_error(const char *prefix, const char *msg, int status)
     int has_tstate_and_gil = (tss_tstate != NULL && tss_tstate == tstate);
     if (has_tstate_and_gil) {
         /* If an exception is set, print the exception with its traceback */
-        if (!_Py_FatalError_PrintExc(fd)) {
+        if (!_Py_FatalError_PrintExc()) {
```

Comment:
Would you mind to pass tss_tstate to _Py_FatalError_PrintExc()? Call the new parameter "tstate".

Reply:
There's already a `tstate` in the function.

    PyThreadState *tstate = _PyThreadState_GET();

Or is your intent that the argument passed in to `_Py_FatalError_PrintExc` would replace that call?

Reply:
> is your intent that the argument passed in to _Py_FatalError_PrintExc would replace that call?

Yes.

Reply:
I've made the changes and updated the branch.